MANIFEST.in

```diff
@@ -0,0 +1,6 @@
+include CHANGELOG.md
+include LICENSE
+recursive-include tests *
```

Comment:
This line can be dropped. The MANIFEST.in I've liked to belongs to a project with a top-level `tests` directory. That's why, in that project, this line is needed. In `ebau-gwr` the tests are all contained in the package that is included anyway.

Reply:
Of course - fixed!